cmd/liqo-controller-manager/main.go

```diff
@@ -378,28 +378,39 @@ func main() {
 		}
 
 		go func() {
-			maxRetries := 5
-			for {
-				// enforce IP remapping for the API Server
-				err := ipamips.EnforceAPIServerIPRemapping(ctx, uncachedClient, *liqoNamespace)
-				if err == nil {
-					break
+			createWithRetry := func(f func() error) {
+				maxRetries := 5
+				for {
+					err := f()
+					if err == nil {
+						break
+					}
+
+					klog.V(3).Infof("Unable to create the IP mapping: %v, retrying...", err)
+
+					maxRetries--
+					if maxRetries == 0 {
+						klog.Errorf("Unable to create the IP mapping: %v", err)
+						os.Exit(1)
+					}
+
+					select {
+					case <-ctx.Done():
+						return
+					case <-time.After(10 * time.Second):
+					}
 				}
+			}
 
-				klog.V(3).Infof("Unable to enforce the API Server IP remapping: %v, retrying...", err)
-
-				maxRetries--
-				if maxRetries == 0 {
-					klog.Errorf("Unable to enforce the API Server IP remapping: %v", err)
-					os.Exit(1)
-				}
+			// enforce IP remapping for the API Server
+			createWithRetry(func() error {
+				return ipamips.EnforceAPIServerIPRemapping(ctx, uncachedClient, *liqoNamespace)
+			})
 
-				select {
-				case <-ctx.Done():
-					return
-				case <-time.After(10 * time.Second):
-				}
-			}
+			// enforce IP remapping for the API Server Proxy
+			createWithRetry(func() error {
+				return ipamips.EnforceAPIServerProxyIPRemapping(ctx, uncachedClient, *liqoNamespace)
+			})
 		}()
```

Comment:
```suggestion
		if err := ipamips.EnforceAPIServerIPRemapping(ctx, uncachedClient, *liqoNamespace); err != nil {
			klog.Errorf("Unable to enforce the API server IP remapping: %v", err)
			os.Exit(1)
		}

		if err := ipamips.EnforceAPIServerProxyIPRemapping(ctx, uncachedClient, *liqoNamespace); err != nil {
			klog.Errorf("Unable to enforce the API server proxy IP remapping: %v", err)
			os.Exit(1)
		}
```

I tried and it works. I think the problem was the webhook, but now we don't have a webhook on IP resources anymore.